README.md

```diff
@@ -106,7 +106,7 @@ $oidc = new OpenIDConnectClient('https://id.provider.com',
                                 'ClientIDHere',
                                 'ClientSecretHere');
 $oidc->setResponseTypes(array('id_token'));
-$oidc->addScope(array('openid'));
+$oidc->addScope(['openid']);
```

Comment:
I prefer the new `[]` brackets style. If we want that it would be nice to update every `array()` to brackets `[]` style.

Reply:
@ricklambrechts no problem, that's done in the README.md file now 👍